redux/userGoalsSlice.ts

```diff
@@ -31,6 +33,12 @@ export const userGoalsSlice: Slice = createSlice({
       }
     },
 
+    isGoalApproaching: (state, action: PayloadAction<boolean>) => {
```

Comment:
i think we should change the name here so it's more obvious that this reducer is a function that sets a value i.e setIsGoalApproaching

when I read isGoalApproaching, i think this is a boolean variable

Reply:
name is changed to setIsGoalApproaching